index.html

```diff
@@ -30,10 +30,10 @@
 
   <!-- Primary Page Layout
   –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-  <div class="container">
-    <div class="row header">
-
-      <div class="one-half column" style="margin-top: 15%">
+  <div class="container ">
```

Comment:
I noticed a few classes have an extra space before the end quote. Is this going to be an issue later when we are trying to identify classes?

Reply:
It shouldn't be. CSS is pretty open about that.